vignettes/bobRoss.Rmd

```diff
@@ -0,0 +1,181 @@
+---
+title: "Bob Ross -Joy of Painting-"
+author: "Jonathan Bouchet"
+date: "`r Sys.Date()`"
+output: 
+rmarkdown::html_vignette:
+df_print: kable
+vignette: |
+  %\VignetteIndexEntry{Comic gender analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+  
+This vignette is based on [538 study : A statistical analysis of the work of Bob Ross](https://fivethirtyeight.com/features/a-statistical-analysis-of-the-work-of-bob-ross/). Bob Ross was an american painter and host of the _The Joy of Painting_, an instructional television program that aired from 1983 to 1994 on PBS in the United States.
+  
+Load required packages to reproduce analysis as well as the dataset.
+
+```{r, message = FALSE, warning = FALSE}
+library(ggplot2)
+library(dplyr)
+library(fivethirtyeight)
+library(ggthemes)
+library(knitr)
+library(corrplot)
+library(ggraph)
+library(igraph)
+library(plyr)
+```
+
+## Data explanation and cleaning
+The author of the article (W. Hickey) went through all Bob Ross's paintings and coded the describing elements (trees, water, mountain,etc ...) : when a element is present in a a painting, it is encoding by `1` in the relevant column.  He wasn't able to analyze 3 paintings. There are also 2 episodes having the same title, so one of them is renamed to avoid errors during a `group_by` episode.
```

Comment:
TYPOS FIXED: 
The author of the article (W. Hickey) went through all Bob Ross's paintings and coded the describing elements (trees, water, mountain, etc ...) : when an element is present in a painting, it is encoding by `1` in the relevant column.  He wasn't able to analyze 3 paintings. There are also 2 episodes having the same title, so one of them is renamed to avoid errors during a `group_by` episode.